RELEASENOTE.md

```diff
@@ -1,8 +1,34 @@
 Version Notes (Current Version: v1.3)
 =======================================
-v1.3 (Pre-release)
+v1.3
 -----------
-
+1. `docfx` improvements
+    1. Add subcommand `docfx template`. You can now `docfx template list` and `docfx template export -A` to list and export all the embeded templates!
```

Comment:
I tried this command and it lists "default". I think op/msdn/vs/docs are also embedded, but didn't show. Not sure whether I understand this command correctly...